netlify.toml

```
@@ -0,0 +1,4 @@
+[build]
+  command = "jlpm install && jlpm build"
```

We would need to run the `inject_api_url.sh` script here as well, allowing us to interact with the API URL once the service is deployed.

```suggestion
  command = "pushd lite && bash ../scripts/inject_api_url.sh && popd && jlpm install && jlpm build"
```